ci/travis_script_python.sh

```diff
@@ -4,28 +4,19 @@ set -e
 
 PYTHON_DIR=$TRAVIS_BUILD_DIR/python
 
+# Re-use conda installation from C++
+export MINICONDA=$TRAVIS_BUILD_DIR/miniconda
+export PATH="$MINICONDA/bin:$PATH"
+export LD_LIBRARY_PATH="$MINICONDA/lib:$LD_LIBRARY_PATH"
```

Comment:
It seems to be required to load libparquet.so: https://travis-ci.org/apache/arrow/jobs/136655629